src/Spryker/Zed/ProductPackagingUnitStorage/Persistence/ProductPackagingUnitStorageRepository.php

```diff
@@ -48,7 +48,6 @@ public function findPackagingProductsByProductAbstractId(int $idProductAbstract)
         $query = $this->getFactory()
```

Comment:
The cause is something else. Products do not have lead products, only their abstract does. Bugfix ticket is created.